test/utils/describeConformanceV5.js

```diff
@@ -28,13 +30,94 @@ function testComponentsProp(element, getOptions) {
   });
 }
 
+/**
+ * Material-UI theme has a components section that allows specifying default props, overrides and variants
+ * Components from @inheritComponent
+ * @param {React.ReactElement} element
+ * @param {() => ConformanceOptions} getOptions
+ */
+function testThemeComponents(element, getOptions) {
+  const render = createClientRender();
+
+  describe('theme: components', () => {
+    it("respect theme's defaultProps", () => {
+      const { muiName, testThemeComponentsDefaultPropWith: testProp = 'id' } = getOptions();
```

Comment:
```suggestion
      const { muiName, testThemeComponentsDefaultPropName: testProp = 'id' } = getOptions();
```
Otherwise it's not clear what "with" refers to: key or value?